repo/packages/J/jenkins/28/config.json

```diff
@@ -0,0 +1,143 @@
+{
+    "type": "object",
+    "properties": {
+        "service": {
+            "description": "Configuration properties for the Jenkins service for DC/OS.",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "The name of the service to display in the DC/OS dashboard.",
+                    "type": "string",
+                    "default": "jenkins"
+                },
+                "cpus": {
+                    "description": "CPU shares to allocate to each Jenkins master.",
+                    "type": "number",
+                    "default": 1.0,
+                    "minimum": 0.1
+                },
+                "mem": {
+                    "description": "Memory (in MB) to allocate to each Jenkins master.",
+                    "type": "number",
+                    "default": 2048.0,
+                    "minimum": 2048.0
+                },
+                "user": {
+                    "description": "The user that the service will run as. In 'strict' security mode, running tasks as root is disabled by default. Ensure that this user exists on each of the agents in your DC/OS cluster.",
+                    "type": "string",
+                    "default": "root"
+                },
+                "marathon-name": {
+                    "description": "the name of the Marathon framework the Jenkins master is deployed from. Change when using a MoM.",
```

Comment:
- Should start with **T**he
- May be add `Marathon on Marathon` for MoM. As a consumer of a publicly available `Jenkins` on `DC/OS`, I may not know what `MoM` stands for ?!